.circleci/config.yml

```diff
@@ -513,7 +513,7 @@ workflows:
           matrix:
             alias: test/e2e
             parameters:
-              k8sVersion: [ v1.19.16-k3s1, v1.21.7-k3s1, kind, kindIpv6 ]
```

Comment:
Should we start using the newer version of K8s? 1.21 -> 1.22/23 ?

Reply:
I'm working on it as we speak, just wanted to see if 1.20 will pass. We spoke with @lahabana to update 1.21 to 1.26 (the latest)